vendors/twtg/drivers/neon-ds-ts-xx-xx/driver.yaml

```diff
@@ -0,0 +1,16 @@
+# Formal driver name
+name: TWTG Neon Temperature Sensor Driver
+# Simple description of the driver functionalities
+description: A driver that decodes and encodes TWTG Neon Temperature Sensor.
+# Mandatory - id of the company that developed the driver
+# This id will be used to construct the id of the driver (Should be in lowercase, no special characters, maximum 8 characters)
+producerId: twtg
+# Email of the developer
+developerEmail: marten.lootsma@twtg.io
```

Comment:
This email should be used like the other twtg drivers: service.devices@twtg.io

Reply:
Corrected